src/App.vue

```diff
@@ -8,16 +8,17 @@ v-app.app
           :oauth="true",
           :forgot-password-url="forgotPasswordUrl")
     v-dialog(v-model="uploader", full-width, max-width="800px")
-      girder-upload(v-if="uploader",
+      girder-upload(
+          v-if="uploadDest",
```

Comment:
@matthewma7 uploadDest is a computed property derived from an asyncComputed property, and so on page load it may be null.  This was not a problem before because by the time uploader toggled on, this computed prop had loaded.  Now the uploader is created on load and must wait.

I'm not sure if we should have the GWC components tolerate invalid props for this reason.  A DataBrowser or Uploader without a valid location is useless, and right now it fills the console with errors.  I think these errors are useful.  

On the other hand, providing the results of asyncComputed properties to GWC components is a very common use case for me so far, so `v-if="locationVariable"` seems to be necessary everywhere I've used GWC.  Are we ok with that?

Reply:
As long as it doesn't break the animation, it is totally ok. What is GWC?